build/build_edge_deb.sh

```diff
@@ -82,6 +85,18 @@ THIS_SCRIPT_DIR=$(dirname "$THIS_SCRIPT")
 XRT_DIR=`readlink -f $THIS_SCRIPT_DIR/../`
 DEBIAN=`readlink -f $THIS_SCRIPT_DIR/debian`
 
+if [ -z $DIST ]; then
+    error "-dist is required option"
+fi
+
+if [[ $DIST == "jammy" ]]; then
+    OS_VERSION="22.04"
+elif [[ $DIST == "focal" ]]; then
+    OS_VERSION="20.04"
+else
+    error "$DIST is not a valid dist option"
```

Comment:
Why? I want my freedom!

Reply:
At present we tested only on these two versions.

Reply:
I agree with keryell's comment. Given the context here that the script is used to create a Debian packages, we should allow the user to specify any possible Debian/Ubuntu distribution which is known to sbuild. So probably we could just delegate the distribution name checking to sbuild.